include/model/Model.h

```diff
@@ -54,7 +54,9 @@ class Model {
     Time delta_t_ = Time(1.0);
     typename ModelVariant::ModelParameters parameters_;
     bool no_self_supply_ = false;
-
+  protected:
+    Time start_time_ = Time(0.0);
+    Time stop_time_ = Time(0.0);
```

Comment:
hmm, we should probably move these into `ModelVariant::ModelParameters` in `Variant.h`